src/index.js

```diff
@@ -8,6 +8,7 @@ const deepmerge = require('deepmerge')
 const fs = require('fs')
 const path = require('path');
 const {cwd} = require('process');
+const { twVar, twProps } = require('./tailwind-css-props');
```

Comment:
Please also add spaces around `cwd` on line 10, or remove the spaces on line 11 :)

Reply:
Me avoiding any extra formating 😅, but consider it done.